tests/App/AppKernel.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SimpleThings\EntityAudit\Tests\App;
```

Comment:
This would be problematic for dev-kit, it wont be able to generate the correct namespace. It will try to use Sonata when adding the console lints.

Since this is tests files, we could change the test namespace to use Sonata and it would fix this problem. Or you will need to introduce an if in the code for this case

Reply:
I've removed the container test in dev-kit https://github.com/sonata-project/dev-kit/pull/1880 in the meantime. 

Maybe the console bin file could read the `AppKernel` class name from an external variable or be able to configure it in dev-kit using as default name the current behaviour.

Reply:
Our goal was stop remove the option from the dev-kit later, so it would be nice to not rely on a name provided by the devkit.